libkbfs/key_manager.go

```diff
@@ -135,25 +126,27 @@ func (km *KeyManagerStandard) getTLFCryptKey(ctx context.Context,
 
 		keys := make([]EncryptedTLFCryptKeyClientAndEphemeral, 0,
 			len(publicKeys))
-		keysInfo := make([]TLFCryptKeyInfo, 0, len(publicKeys))
+		serverHalfIDs := make([]TLFCryptKeyServerHalfID, 0, len(publicKeys))
 		publicKeyLookup := make([]int, 0, len(publicKeys))
 
 		for i, k := range publicKeys {
-			info, ok, _ := md.GetTLFCryptKeyInfo(keyGen, uid, k)
-			if ok {
-				ePublicKey, err := md.GetTLFEphemeralPublicKey(keyGen, uid, k)
-				if err != nil {
-					continue
-				}
-
-				keysInfo = append(keysInfo, info)
-				keys = append(keys, EncryptedTLFCryptKeyClientAndEphemeral{
-					PubKey:     k,
-					ClientHalf: info.ClientHalf,
-					EPubKey:    ePublicKey,
-				})
-				publicKeyLookup = append(publicKeyLookup, i)
+			ePublicKey, encryptedClientHalf, serverHalfID, found, err := kmd.GetTLFCryptKeyInfo(keyGen, uid, k)
```

Comment:
it's not really returning a `TLFCryptKeyInfo` now, maybe we should rename the method?